doc/packer.txt

```diff
@@ -328,7 +295,7 @@ update()                                                        *packer.update()
     such as `update({preview_updates = true}, ...)` to preview potential
     changes before updating (same as `PackerUpdate --preview`).
 
-use()                                                              *packer.use()*
+add()                                                              *packer.use()*
     `use` allows you to add one or more plugins to the managed set. It can be
     invoked as follows:
     - With a single plugin location string, e.g. `use <STRING>`
```

Comment:
```suggestion
add()                                                              *packer.add()*
    `add` allows you to add one or more plugins to the managed set. It can be
    invoked as follows:
```